### PR TITLE
Bug fix with color column in helper list

### LINKED ISSUE #23638: Bug with color column
#### Describe the bug

When you making fields_list in controller, you can to add property color, where the name of key in array. When you have key in array as "color", it is ok, but when you have seting other value, will be bug. 

1.  Make simple controller
2. Add to fields_list column with color, where value != color
3. Test it

#### Additional information

* PrestaShop version: 1.7.6.0
* PHP version: 7.2

### PR DESCRIPTION
**Pull request by fest73**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Bug fix with color column in helper list
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes https://github.com/PrestaShop/PrestaShop/issues/23638
| How to test?      | I has been writing here: https://github.com/PrestaShop/PrestaShop/issues/23638
| Possible impacts? | -

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/23634)

### CODE REVIEW

**Review by NeOMakinG — APPROVED** [review]
LGTM

**Review by PierreRambaud — APPROVED** [review]

**Comment by Robin-Fischer-PS** [comment]
Hi everyone,  

Since the issue linked to the PR has not been reproduced yet, this PR is **not** ready for QA.

@dariusakafest Please check your issue and help us reproduce the bug by providing clearer steps (as @hibatallahAouadni asked ), until that this PR will not be tested and merged.

**Comment by matks** [comment]
> @dariusakafest Please check your issue and help us reproduce the bug by providing clearer steps (as @hibatallahAouadni asked ), until that this PR will not be tested and merged.

Unfortunately the issue could not be confirmed.

This PR is stale since March.

I close this PR, following our [review guidelines](https://devdocs.prestashop.com/1.7/project/maintainers-guide/reviewing-pull-requests/) "Pull Requests may be closed after 30 days of inactivity". If however you are still interested in it, please ping us :) we can reopen.

Thank you for your contribution.

**Pull request closed without merging**
